frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.tsx

```diff
@@ -138,6 +138,18 @@ export const ParametersWidgetContainer = styled(FullWidthContainer)<{
     `}
 `;
 
+export const FIXED_WIDTH = "1048px";
+export const FixedWidthContainer = styled.div<{
+  isFixedWidth: boolean;
+}>`
+  margin: 0 auto;
+  ${({ isFixedWidth }) =>
+    isFixedWidth &&
+    css`
+      max-width: ${FIXED_WIDTH};
+    `}
+`;
+
```

Comment:
Create the fixed width container and use `isFixedWidth` variable to dynamically set the width